lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -721,7 +721,7 @@ def cmd_history_help
 
   def cmd_history_tabs(str, words)
     return [] if words.length > 1
-    @@history_opts.fmt.keys
+    @@history_opts.option_names
```

Comment:
Not strongly opinionated here, but maybe `keys`? 🤔 

```suggestion
    @@history_opts.option_keys
```